runtime/lua/vim/_editor.lua

```diff
@@ -1048,7 +1049,7 @@ function vim.deprecate(name, alternative, version, plugin, backtrace)
   -- e.g., when planned to be removed in version = '0.12' (soft-deprecated since 0.10-dev),
   -- show warnings since 0.11, including 0.11-dev (hard_deprecated_since = 0.11-dev).
   if plugin == 'Nvim' then
-    local current_version = vim.version() ---@type Version
+    local current_version = vim.version() ---@type vim.Version
```

Comment:
Should we maybe add (and is it worth it) a CI check to ensure all Lua types are namespaced correctly?

Reply:
Yes, though we can't do that until all the types are correctly named first.